### ISSUE TITLE
Compile time increased significantly when upgrading from Deno@v1.0.5 to Deno@v1.1.0

### ISSUE THREAD

**Issue by wongjiahau** [denoland/deno#6287]
## Steps to reproduce
1. Create a file named `main.ts` with the following content:
```ts
import { LevoApp } from "https://deno.land/x/levo@v0.0.8/mod/levo-app.ts";
console.log('hello')
LevoApp.start({
  port: 5000,
  minifyJs: true,
  cachePages: true,
  rootDir: new URL("./root", import.meta.url),
});
```
2. Run the following command to download dependencies, such that the benchmark duration won't include the time consumed on pulling dependencies:
```
deno run --allow-all --unstable main.ts
```
3. IMPORTANT. Replace the argument of `console.log` with another 5-character string in `main.ts`, this step is necessary such that when we run the benchmark, Deno won't load from cache.
4. Upgrade to Deno v1.0.5
```
deno upgrade --version=1.0.5
```
5. Benchmark the compile time.
```
time deno run --allow-all --unstable main.ts
```
6. Upgrade to Deno v1.1.0
```
deno upgrade --version=1.1.0
```
7. Run step 3.
8. Benchmark the compile time.
```
time deno run --allow-all --unstable main.ts
```
9. Re-run step 3 to step 8, but instead of starting with Deno v1.0.5, start with Deno v1.1.0. 

## Expected result
The compile time difference between the two versions of Deno shouldn't be significant.

## Actual result
The compile time of Deno v1.1.0 is significantly (more than 200%) longer than that of Deno v1.0.5.
The following is the result I obtained:
```
// Test with Deno v1.0.5 first
Deno@1.0.5    3.69 real         7.00 user          0.29 sys
Deno@1.1.0    11.13 real        19.26 user         0.48 sys

// Test with Deno v1.1.0 first
Deno@1.1.0    11.50 real        20.28 user         0.54 sys
Deno@1.0.5    4.00 real         7.03 user           0.28 sys
```

## Enviroment
- fish shell
- macOS Catalina (version 10.15.4 (19E266))
- memory: 16 GB 2400 MHz DDR4
- processor: 2.2 GHz 6-Core Intel Core i7

**Comment by caspervonb** [comment]
I came up with slightly different results

| Version | Real      | User      | System   |
|---------|-----------|-----------|----------|
| 1.1.0   | 0m22.477s | 0m32.524s | 0m0.800s |
| 1.0.5   | 0m28.186s | 0m3.628s  | 0m0.297s |

**Comment by wongjiahau** [comment]
@caspervonb This is the benchmark I got instead.
```
Deno@v1.1.0 
11.20 real        19.30 user         0.51 sy

Deno@v1.0.5
4.04 real         6.91 user         0.27 sys
```

**Comment by wongjiahau** [comment]
Seems to be fixed by Deno@v1.1.1
